svc-managers/mgrmodel/manager_test.go

```diff
@@ -61,7 +61,7 @@ func TestManager_Update(t *testing.T) {
 			"State": "Absent",
 		},
 	}
-	err = UpdateManagersData(key, m)
+	err = UpdateManagersData(key, m, "Managers")
 
 	data, err := GetResource(table, key)
```

Comment:
This method GetResource only returns "body" if key is /redfish/v1/Managers/uuid1:1/Ethernet. So what are we testing here ??

Reply:
GetResource is getting the data from the key and later in the code cross checking if the data got updated or not

Reply:
I meant mockGetResource  which is called when we call GetResource - check the definition and see if I am making sense

Reply:
Updated the mockGetResource